src/service/dealService.ts

```diff
@@ -280,6 +282,71 @@ const createCoupangImage = async (req, res, next) => {
   }
 };
 
+const readDealDetail = async (req, res, next) => {
```

Comment:
```suggestion
const readDealDetail = async (req: Request, res: Response, next: NextFunction) => {
```
과 같이 명시적으로 타입을 표기해주는게 좋을 것 같습니다!